linera-base/src/data_types.rs

```diff
@@ -1244,28 +1245,112 @@ impl CompressedBytecode {
     }
 }
 
+impl Serialize for BlobContent {
```

Comment:
Do we need this boilerplate?

Reply:
Did you miss this ?
https://serde.rs/feature-flags.html#rc

Reply:
Yes, thank you.